pkg/cmd/config.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func isValidKey(key string) bool {
+	switch key {
+	case
+		"user.name",
+		"user.email":
+		return true
+	}
+	return false
+}
+
+func NewCmdConfig(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config name [value]",
+		Short: "Get or set global options",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Nothing else was returning an error at any point so I could just remove that line in https://github.com/ClusterHQ/dvol/pull/63/commits/f8b3fe7892e6aa9d07753dfbd590847d0931168f and all the tests still pass.